v3/src/components/inspector-panel.tsx

```diff
@@ -53,3 +54,43 @@ export const InspectorMenu = ({children, icon, tooltip, testId, onButtonClick}:I
     </Menu>
   )
 }
+
+interface IInspectorPalette {
+  children: ReactNode
+  Icon?: ReactNode
+  title?: string
+  paletteTop?: number
+}
+
+export const InspectorPalette =({children, Icon, title, paletteTop}:IInspectorPalette) => {
+  const paletteRef = useRef(null)
+  const [paletteHeight, ] = useState(251)
```

Comment:
If this is never changed, why is it in state? Also, it doesn't appear to be the correct value given that the same value in the .scss file doesn't appear to encompass all of the content.